lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -96,7 +96,7 @@ <h3 class="hd hd-3">${_("Reports Available for Download")}</h3>
   %endif
 
     ## Translators: a table of URL links to report files appears after this sentence.
-    <p>${_("<b>Note</b>: To keep student data secure, you cannot save or email these links for direct access. Copies of links expire within 5 minutes.")}</p><br>
+    <p>${HTML(_("<b>Note</b>: To keep student data secure, you cannot save or email these links for direct access. Copies of links expire within 5 minutes."))}</p><br>
```

Comment:
This should use `Text()` with the HTML (e.g. `<b>`) interpolated in.  The reason for this is that it is possible that the translated text could have unexpected HTML characters in the translation that would need to be escaped.